LEGO1/mxioinfo.h

```diff
@@ -1,10 +1,22 @@
 #ifndef MXIOINFO_H
 #define MXIOINFO_H
 
+#include "windows.h"
```

Comment:
Including `windows.h` is known to cause issues because of their macros. Instead, include `legoinc.h` which handles the macros automatically (there should be a CI test at some point to verify this)